yarn.lock

```diff
@@ -3127,22 +3109,22 @@
     "@polymer/iron-iconset-svg" "^3.0.0"
     "@polymer/polymer" "^3.0.0"
 
-"@vaadin/vaadin-material-styles@^1.2.0", "@vaadin/vaadin-material-styles@^1.3.2":
+"@vaadin/vaadin-material-styles@^1.1.0", "@vaadin/vaadin-material-styles@^1.2.0", "@vaadin/vaadin-material-styles@^1.3.2":
   version "1.3.2"
   resolved "https://registry.yarnpkg.com/@vaadin/vaadin-material-styles/-/vaadin-material-styles-1.3.2.tgz#d2c1bd290db16721152ae672dbe052c381686696"
   integrity sha512-EFrvGScoxhLNrPnWtT2Ia77whjF2TD4jrcyeh1jv9joCA2n5SUba+4XJciVSGmopqqQato6lwRnZSvMLJX7cyw==
   dependencies:
     "@polymer/polymer" "^3.0.0"
 
-"@vaadin/vaadin-ordered-layout@^2.0.0-alpha1":
-  version "2.0.0-alpha1"
-  resolved "https://registry.yarnpkg.com/@vaadin/vaadin-ordered-layout/-/vaadin-ordered-layout-2.0.0-alpha1.tgz#74ab61ece84b05e56ddfef496d0282b8be17afd1"
-  integrity sha512-JNouDr+oIeIhnnbhXRFVqnWStikmxR+RhbOXdIGb8t076cN94ckzQf2XAE+B4TglxFMIhZSs6v/Ug0KbtZ9zWg==
+"@vaadin/vaadin-ordered-layout@^1.4.0":
```

Comment:
Avoid reverting vaadin-ordered-layout alpha upgrade.

Reply:
Did you reset your branch to my last push state, or rebase?

Reply:
Why do you ask? I think you can see the commits history on fork's branch as a `collaborator`.
Don't have an access to the `merge commit` details here: https://github.com/dominik-stypula-polcode/aybolit/commit/d7cd276a5897b0bcba319e0fdf37bef2f1d7ad39?

Reply:
My last clean push resulted in https://github.com/dominik-stypula-polcode/aybolit/commits/4e2faf4f6a22facebd305bcbcc511009451090d3 state.

Now this PR is 18 commits again, with outdated commits all back.